packages/turf-collect/.npmignore

```diff
@@ -1,2 +1,3 @@
 test
-coverage
```

Comment:
This folder is created when coverage tests are ran, you shouldn't be removing this.